### PR TITLE
Amib1 ee 333 extract auth function

### PR DESCRIPTION
**Pull request by amaanibn-nasar1-nhs**

### CODE REVIEW

File: src/mongodb/mongo.ts [hunk 1]
```diff
@@ -4,7 +4,13 @@ import { LambdaLogger } from "../logger";
 
 const logger = new LambdaLogger("ee-utils/mongodb", logReferences);
 
-export const connectToDatabaseViaEnvVar = async (): Promise<void> => {
+export const connectToDatabaseViaEnvVar = async (
+  options = {
+    ssl: true,
+    tlsCAFile: `${__dirname}/global-bundle.pem`,
+    retryWrites: false,
```
Comment (regularfry) [inline src/mongodb/mongo.ts:11]: Is there a reason we're switching retries off?  I can see this is inherited from the older code but I don't know why we want to disable an error correction mechanism here.

**Review by regularfry — CHANGES_REQUESTED** [review]
Just one thing I'm not sure about.

File: src/mongodb/mongo.ts (hunk 1 above)
Comment (amaanibn-nasar1-nhs) [reply src/mongodb/mongo.ts:11]: I don't actually know, I can't find it as an option here https://mongoosejs.com/docs/connections.html#options

Comment (regularfry) [reply src/mongodb/mongo.ts:11]: Yeah, as far as I could tell by looking at it it's either ignored or it's [this](https://www.mongodb.com/docs/manual/core/retryable-writes).  But even then I'm having a really hard time understanding what it actually *does*.

Comment (amaanibn-nasar1-nhs) [reply src/mongodb/mongo.ts]: I've decided to just remove this now

**Review by regularfry — DISMISSED** [review]

**Pull request merged**